lib/gui-utils/src/GUIUtils.hpp

```diff
@@ -43,7 +43,7 @@ class GUIUtils
 
   void displaySensorAvarage(int avarage);
 
-  void displaySensorData(int pm25, int pm10);
+  void displaySensorData(int pm25, int pm10, int chargeLevel);
```

Comment:
is possible maybe that we need a different prototype of this method, also we need using preprocessor directives in the contents of this method